charts/internal/machineclass/templates/machineclass.yaml

```diff
@@ -24,6 +24,7 @@ metadata:
 {{- end }}
 {{- if $machineClass.nodeTemplate }}
 nodeTemplate:
+  architecture: {{$machineClass.nodeTemplate.architecture}}
```

Comment:
```suggestion
  architecture: {{ $machineClass.nodeTemplate.architecture }}
```

Reply:
Nit: make the change use the same formatting as the rest of the file

Reply:
done